platform/centec/one-image.mk

```diff
@@ -3,6 +3,16 @@
 SONIC_ONE_IMAGE = sonic-centec.bin
 $(SONIC_ONE_IMAGE)_MACHINE = centec
 $(SONIC_ONE_IMAGE)_IMAGE_TYPE = onie
-$(SONIC_ONE_IMAGE)_INSTALLS += $(CENTEC_SDK_KERNEL)
+$(SONIC_ONE_IMAGE)_INSTALLS += $(CENTEC_E582_48X6Q_PLATFORM_MODULE)
 $(SONIC_ONE_IMAGE)_DOCKERS += $(SONIC_INSTALL_DOCKER_IMAGES)
+
+ifeq ($(SONIC_CONFIG_DEBUG),y)
+GDB = gdb
+VIM = vim
+OPENSSH = openssh-client
+SSHPASS = sshpass
+STRACE = strace
+$(SONIC_ONE_IMAGE)_DBG_PACKAGES += $(GDB) $(VIM) $(OPENSSH) $(SSHPASS) $(STRACE)
+endif
```

Comment:
this is not centec platform specific. Can you remove from this PR. You can create a separate PR this DEBUG feature.

Reply:
thanks, i will close this PR and create a new PR after these changes are done.